INSTALLATION.md

```diff
@@ -1,29 +1,128 @@
 # Installation
 
-The latest stable version of _taipy-gui_ is available through _pip_:
+There are different ways to install Taipy GUI, depending on how
+you plan to use it.
+
+If your goal is to look into the code, modify and improve it, go straight
+to the [source installation](#installing-for-development) section.
+
+Taipy GUI needs your system to have Python 3.8 or above installed.
```

Comment:
Taioy GUI relies on Python 3.8 or above ?

Reply:
Well... that's what the setup.py says: `python_requires=">=3.8"`
As well as the doc and the Web site...